test/diff/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(diff_files/diff_test_files_autogen.cmake)
+
+add_spvtools_unittest(TARGET lcs
+  SRCS lcs_test.cpp
+  LIBS SPIRV-Tools-diff
+)
+
+add_spvtools_unittest(TARGET diff
+  SRCS diff_test.cpp diff_test_utils.h diff_test_utils.cpp
+       ${DIFF_TEST_FILES} ${CMAKE_SOURCE_DIR}/tools/util/cli_consumer.cpp
```

Comment:
You cannot use `CMAKE_SOURCE_DIR` here.  If spirv-tools is not the top level project, then this will not point to the correct directory.

Reply:
You want to use `spirv-tools_SOURCE_DIR` instead.  This should fix the shaderc smoke test.

Reply:
Thanks, I'll give it a shot!